action.yml

```diff
@@ -36,9 +36,9 @@ inputs:
 runs:
   using: "composite"
   steps:
-      - name: Build Vue
+      - name: Build Vue MohaElder
```

Comment:
Why the name change?

Reply:
oops I’m so sorry i mistakenly pushed this change. I changed it to test if the action gets recognized. I will close the PR now!